docs/reference/experiments.md

```diff
@@ -8,13 +8,14 @@ The following is a list of currently available experiments. We'll remove experim
 
 * `bolt-install`: enables creating, installing, and running Bolt projects that manage their app manifest on app settings (remote manifest).
     * `slack create` and `slack init` now set manifest source to "app settings" (remote) for Bolt JS & Bolt Python projects ([PR#96](https://github.com/slackapi/slack-cli/pull/96)).
+    * `slack run` and `slack install` support creating and installing Bolt Framework apps that have the manifest source set to "app settings (remote)" ([PR#111](https://github.com/slackapi/slack-cli/pull/111), [PR#TODO](https://github.com/slackapi/slack-cli/pull/TODO)).
```

Comment:
note: I added the `slack run` in PR #111 but it may have been removed by a rogue docs sync 🥷